src/game.cpp

```diff
@@ -6875,17 +6875,23 @@ bool game::summon_mon( const std::string id, const tripoint &p )
     // Set their last upgrade check to the current day.
     mon.reset_last_load();
```

Comment:
Is this last load thing used for anything else? It seems that it was used to make the upgrades not happen more than once a day.

If it's not used for anything else some clean up is in order.